test/mocha.opts

```diff
@@ -1,3 +1,3 @@
+--require should
 --ui tdd
 --timeout 30s
```

Comment:
You should definitely add `--require test/babel-register.js`

And add a file like this:
https://github.com/taskcluster/taskcluster-queue/blob/rewrite-on-azure-storage-services/test/6to5-register.js

So that everything will be compiled on the fly....Otherwise tests won't work when you start using babel.